gtk/gtk.go

```diff
@@ -1232,7 +1232,11 @@ func (v *Dialog) GetWidgetForResponse(id int) *Widget {
 }
 
 // gtk_dialog_get_action_area
-// gtk_dialog_get_content_area
+
+func (v *Dialog) GetContentArea() *VBox {
+	return v.GetVBox()
```

Comment:
Why you don't call gtk_dialog_get_content_area ?

Reply:
it's same
```
GtkWidget *
gtk_dialog_get_content_area (GtkDialog *dialog)
{
  g_return_val_if_fail (GTK_IS_DIALOG (dialog), NULL);

  return dialog->vbox;
}
```

Reply:
I'll rewrite with gtk_dialog_get_content_area